frontend/public/locales/en/translation.json

```diff
@@ -2,6 +2,7 @@
     " No status: ": " No status: ",
     " With violations: ": " With violations: ",
     " Without violations: ": " Without violations: ",
+    "-----BEGIN CERTIFICATE-----\n<MY_TRUSTED_CA_CERT>\n-----END CERTIFICATE-----": "-----BEGIN CERTIFICATE-----\n<MY_TRUSTED_CA_CERT>\n-----END CERTIFICATE-----",
```

Comment:
I am not sure if it makes sense to translate this. The header and footer strings must not be translated, so it is just the MY_TRUSTED_CA_CERT string that is a candidate for translation.